hubbado-idempotence-sequential-fixtures.gemspec

```diff
@@ -0,0 +1,24 @@
+Gem::Specification.new do |spec|
+  spec.name     = "hubbado-idempotence-sequential-fixtures"
+  spec.version  = "1.0.0"
+  spec.authors  = ["Alexander Repnikov", "Sam Stickland"]
+  spec.email    = ["aleksander.repnikov@gmail.com", "sam@hubbado.com"]
+
+  spec.summary  = "TestBench fixtures for hubbado-idempotence-sequential"
+  spec.homepage = "https://www.github.com/hubbado/hubbado-idempotence-sequential-fixtures"
+  spec.license  = "MIT"
+  spec.required_ruby_version = Gem::Requirement.new(">= 3.3")
+
+  spec.metadata["homepage_uri"] = spec.homepage
+  spec.metadata["source_code_uri"] = spec.homepage
+
+  spec.require_paths = ["lib"]
+  spec.files = Dir.glob('{lib}/**/*')
+
+  spec.add_dependency 'evt-entity_projection-fixtures'
+  spec.add_dependency 'evt-messaging-fixtures'
+  spec.add_dependency 'hubbado-idempotence-sequential'
+
+  spec.add_development_dependency "hubbado-style"
+  spec.add_development_dependency "test_bench"
```

Comment:
If you going to make the gem public, this probably is not reachable from internet.

I guess the problem is that
`https://github.com/hubbado/hubbado-idempotence-sequential` 
and
`https://github.com/hubbado/hubbado-style`
are not public (?)

In general, I was not able to fetch and install this on my machine...

Reply:
hubbado-idempotence-sequential want to make it public but not sure about hubbado-style.

Reply:
I'm not sure, does it it need hubbado-style

Reply:
Let's make hubbado-style public?

Reply:
@AlfonsoUceda @arepnikov I've made hubbado-style public

Reply:
but `hubbado-idempotence-sequential` is still private and is under "https://rubygems.pkg.github.com/hubbado", so I can't install gems...

Reply:
@samstickland ⬆️

Reply:
@arepnikov Oh! I've made it public now

Reply:
OK, still have problem with install gems, but maybe it's on my side. Will double check it later.

I posted more details on our Slack general channel.

@samstickland